spec/index.bs

```diff
@@ -2733,6 +2757,79 @@ Given a {{GPUCommandEncoder}} |encoder| and the arguments {{GPUTextureCopyView}}
+  multiple continuous {{GPUTexture}} [=subresources=].
+
+  |source| and |copySize| define the region of the source texture [=subresources=].
+
+  |destination| and |copySize| define the region of the destination texture [=subresources=].
+
+</div>
+
+<div algorithm class=validusage>
+
+<dfn abstract-op>copyTextureToTexture Valid Usage</dfn>
+
+Given a {{GPUCommandEncoder}} |encoder| and the arguments {{GPUTextureCopyView}} |source|,
+{{GPUTextureCopyView}} |destination|, {{GPUExtent3D}} |copySize|, let
+
+  - A |copy of the whole subresource| be the command |encoder|.{{GPUCommandEncoder/copyTextureToTexture()}} whose parameters |source|, |destination| and |copySize| meet the following conditions:
+    - The [=textureCopyView subresource size=] of |source| must be equal to |copySize|.
+    - The [=textureCopyView subresource size=] of |destination| must be equal to |copySize|.
+
+The following validation rules apply:
+
+  For |encoder|:
+  - |encoder|.{{GPUCommandEncoder/copyTextureToTexture()}} must not be called when a {{GPURenderPassEncoder}}
+    is active on |encoder|.
+  - |encoder|.{{GPUCommandEncoder/copyTextureToTexture()}} must not be called when a {{GPUComputePassEncoder}}
+    is active on |encoder|.
+
+  For |source|:
+  - |source| must be [=valid=].
+  - |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[textureUsage]]}} must contain
+    {{GPUTextureUsage/COPY_SRC}}.
+
+  For |destination|:
+  - |destination| must be [=valid=].
+  - |destination|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[textureUsage]]}} must contain
+    {{GPUTextureUsage/COPY_DST}}.
+
+  For the texture {{GPUTexture/[[sampleCount]]}}:
+  - |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[sampleCount]]}} must be equal to |destination|.
+    {{GPUTextureCopyView/texture}}.{{GPUTexture/[[sampleCount]]}}.
+  - If |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[sampleCount]]}} is greater than 1:
+    - The copy with |source|, |destination| and |copySize| must be a |copy of the whole subresource|.
+
+  For the texture {{GPUTexture/[[format]]}}:
+  - |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}} must be equal to |destination|.
+    {{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}}.
+  - If |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}} is a depth-stencil format:
+    - The copy with |source|, |destination| and |copySize| must be a |copy of the whole subresource|.
+
+  For the copy ranges:
+  - [=Valid Texture Copy Range=] applies to |source| and |copySize|.
+  - [=Valid Texture Copy Range=] applies to |destination| and |copySize|.
+  - If |source|.{{GPUTextureCopyView/texture}} is the same as |destination|.{{GPUTextureCopyView/texture}}:
+    - Either |source|.{{GPUTextureCopyView/arrayLayer}} is not equal to |destination|.{{GPUTextureCopyView/arrayLayer}} or |source|.{{GPUTextureCopyView/mipLevel}} is not equal to |destination|.{{GPUTextureCopyView/mipLevel}}.
```

Comment:
nit: can this say something like "the source subresource must not equal the destination subresource"